### DISCUSSION TITLE
Is it possible using Template variable in alert image screenshot?

### DISCUSSION THREAD

**Discussion by Yeongun** [grafana/grafana#51759] [Alerting]
Hello, I am very satisfied with the alerting function of Grafana9, but I have a suggestion about image screenshot..

In my estimation,  **pkg/services/ngalert/image/service.go**  there is only two option for make screenshot. ( Dashboard UID, Panel UID ) 

I created alert rule by using multidimensional expression, and send image screenshot to my Slack Channel. 

Although it is a single panel, the details vary depending on the template variable, but only the image sent is transferred with the default variable in the dashboard.

In addition to Dashboard UID and Panel UID, it seems necessary to add Template Variable as an option to resolve this issue.

Thank you!

https://community.grafana.com/t/does-the-grafana-9-alert-have-an-annotation-for-template-variables-image-rendering/68179

**Comment by tim-felix-lange** [comment]
Having the same issue. We're monitoring a product with distinction into different customer's data. Previously we've split this up into multiple dashboards, which was always very tedious to keep in sync when being updated. 
With the advent of Unified Alerting we would like to consolidate everything into a single dashboard where we can select the customer via template variable. While this works fine with respect to the dashboard and the alerting, the image renderer seems to be stuck in the pre-ngalert era :-(

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
